src/packages/tsconfig.json

```diff
@@ -1,26 +0,0 @@
-{
-    "extends": "../../settings.tsconfig.json",
-    "include": ["*"],
```

Comment:
Understand about not needing it on a 'parent' project - however I thought it was 100% needed on the actual projects, noticed you deleted includes from each projects tsconfig as well. What if I want to exclude certain files in the projects? (util, user1, 2 etc) - such as jest test files / other misc.

Reply:
Sorry, my comment didn't really make sense - let me start fresh.

If you're making a "parent" (docs usually say "solution") project, you want `files:[]` to prevent any files from being included directly in that project.  (It's not technically wrong to have files in it, but the language service recognizes solution projects using a heuristic and you might lose functionality.)

If you're making a "leaf" project, you do want to use `include` or `files`, but the default value of `include` is frequently fine.  (You're making me second guess myself - I guess if you remove it and you get an error, put it back.  Again, it should be there - I was only removing values that looked like the default.)